Tests/NostrSDKTests/RelayRequestEncodingTests.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  RelayRequestEncodingTests.swift
+//  
+//
+//  Created by Joel Klabo on 5/25/23.
+//
+
+@testable import NostrSDK
+import XCTest
+
+final class RelayRequestEncodingTests: XCTestCase, FixtureLoading {
+
+    func testEncodeClose() throws {
+        guard let request = RelayRequest.close(subscriptionId: "some-subscription-id") else {
```

Comment:
There's a better way to do this in tests:
```swift
let request = try XCTUnwrap(RelayRequest.close(subscriptionId: "some-subscription-id"), "failed to encode request")
```

Reply:
Yeah, that's much nicer 🤙 fixed: fa1b42e